app/views/coronavirus_form/transport_type.html.erb

```diff
@@ -27,5 +28,15 @@
     }
   end
 } %>
+
+<%= render "govuk_publishing_components/components/textarea", {
```

Comment:
@bilbof has an open PR to cache some of these, if that gets merged first we should adjust these here too?